recipes/libjpeg-turbo/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.5.1" %}
+{% set libjpeg_version = "8c" %}
+
+package:
+    name: libjpeg-turbo
+    version: {{ version }}
+
+source:
+    fn: libjpeg-turbo-{{ version }}.tar.gz
+    url: http://downloads.sourceforge.net/project/libjpeg-turbo/{{ version }}/libjpeg-turbo-{{ version }}.tar.gz
+    md5: 55deb139b0cac3c8200b75d485fc13f3
+
+build:
+    number: 0
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
+        
+requirements:
+    build:
+        - python  # [win]
```

Comment:
By the way, I used [the wiki](https://github.com/conda/conda/wiki/VC-features) for extra help here. If the `vc` package needs to be always be added as a run-time dependency, then you might want to add that extra detail there as well.